src/installer/pkg/projects/host-packages.proj

```diff
@@ -1,9 +1,6 @@
 <Project Sdk="Microsoft.Build.Traversal" DefaultTargets="Pack">
   <ItemGroup>
     <ProjectReference Include="Microsoft.NETCore.DotNetAppHost\Microsoft.NETCore.DotNetAppHost.pkgproj" />
```

Comment:
That package is still needed, right? It would be amazing if we would convert it from a pkgproj to a normal .proj and use NuGet's Pack task, so that we can remove all the pkgproj infrastructure from src/installer/*. As part of that, we should probably also move the package wrapper project to the host directory.

pkgprojs are deprecated and we want to eventually remove all of them from dotnet/runtime.

Reply:
Yeah, this one has binaries folks would use for custom hosting scenarios. I imagine we could/should convert it - that said, I'm not familiar with the differences between pkgproj / regular NuGet Pack.

I think the pkgproj infrastructure in src/installer is also used by Microsoft.DotNet.ILCompiler though: https://github.com/dotnet/runtime/blob/76d17b25252ce14b7e36c2e6854fe416db60f5cf/src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj